cardano-launcher/src/Cardano/Shell/Launcher.hs

```diff
@@ -46,9 +56,12 @@ newtype UpdateRunner = UpdateRunner { runUpdate :: IO ExitCode }
 
 newtype LauncherRunner = LauncherRunner { runLauncher :: IO ExitCode }
```

Comment:
I've overlooked this in the previous PR but, I think the naming is a bit misleading. We're only re-running part of the launcher code (which is `runWallet`) and not the whole launcher process.

Reply:
LGTM on everything else. Well done.

Reply:
Good catch. Fixed.